src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-SPECVIEW-AMD.json

```diff
@@ -0,0 +1,52 @@
+{
+  "Description": "SPEC view GPU Performance Workload",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "01:00:00",
+    "SupportedPlatforms": "win-x64",
+    "SupportedOperatingSystems": "Windows"
+  },
+  "Parameters": {
+    "GpuModel": "v620"
+  },
+  "Actions": [
+    {
+      "Type": "SpecViewExecutor",
+      "Parameters": {
+        "Scenario": "SPECTestingAllViewSets",
+        "CommandArguments": "-viewset \"3dsmax,catia,creo,energy,maya,medical,snx,sw\" -nogui",
```

Comment:
How long does it generally run with all scenarios? Does it make sense to separate?

Reply:
Good point. I will make viewset a cmdline argument. Users can specify which tests they would like specviewperf to run.

Reply:
So, if all these scenarios runs on every chip, and just takes 5 minutes, we can just make it default, like SPECcpu, we just run everything. But if that's not the case please let me know and decide on what you think is best

Reply:
Each scenario takes about 5 minutes to run. Running all the scenarios takes 30-40 minutes. I think it's better if users can decide which viewset(s) they would like to run.